### ISSUE TITLE
✔️ 🚀 Feature Request: Horizontal scrolling containers

### ISSUE THREAD

**Issue by 93petter93** [elementor/elementor#15765]
Hi! It would be amazing if you could make a horizontal scrolling section or/and widget. Could be used for products, categories, posts, images, menus, filters and more... Something like the one at Netflix.

Example:
![1](https://user-images.githubusercontent.com/58670859/126932680-7311997d-ae94-40bb-89fb-59ca2cb1e495.gif)
![image](https://user-images.githubusercontent.com/58670859/126933046-30d6f40b-4802-4ddf-9a23-9f2fb92f04f0.png)
![image](https://user-images.githubusercontent.com/58670859/126933060-92fb14e1-c62d-47ed-ba86-2e88960bbb8a.png)
![image](https://user-images.githubusercontent.com/58670859/126933105-c6589803-e3f3-471b-a6c1-d1ad9b64ddf1.png)

**Comment by 79ho3ein** [comment]
+1

**Comment by paauserrat** [comment]
Hi, I was going to post this issue but I found this thread.

My suggestion is to add horizontal and vertical containers that make anything that's inside them scrollable. This will allow us to have our custom galleries as well as scroll everything.

The scrolling container can be toggled by activating a new "make container scrollable" option for each. Then, new style and customization options should be accessible:
- show or hide scroll pointers
- customize pointers
- autoscroll
- px per scroll
- stop or not on hover or interact
Basically some options that the gallery already has, but to be able to create our own scrollers.

**Comment by nicholaszein** [comment]
Hello there,

thanks for submitting your feature request, and for being so patient! 🙏 

### We have great news! 🙌 

Part of your request was already solved. Since Elementor Pro v3.11 you can now use Loop Carousels to display your posts and scroll them horizontally!! 🥳 

✔️ Feel free to check it out, and update to the latest version.

As for the other elements, like Nested Carousel, these are planned, and you can follow our Roadmap [here](https://elementor.com/roadmap) to check when they will be released.

Please understand that while we do welcome new feature requests, we can't track multiple requests per thread. For this reason, I am closing this issue as Shipped. 🚀 

Best regards
